vowpalwabbit/explore_eval.cc

```diff
@@ -272,10 +222,13 @@ base_learner* explore_eval_setup(arguments& arg)
   arg.all->p->lp = CB::cb_label;
   arg.all->label_type = label_type::cb;
 
-  learner<explore_eval>& l = init_learner(data, base, predict_or_learn<true>, predict_or_learn<false>, 1, prediction_type::action_probs);
+  learner<explore_eval>& l = init_learner(data, base, 
+    do_actual_learning<true>, do_actual_learning<false>, 
+    1, prediction_type::action_probs);
 
   l.set_finish_example(finish_multiline_example);
   l.set_finish(finish);
   l.set_end_examples(end_examples);
+  l.set_test_example(CB::example_is_test);
```

Comment:
This feels like it's a property of the label type rather than a property of the reduction.

Reply:
Defered to John's upcoming change.